src/blitter/32bpp_anim_sse4.cpp

```diff
@@ -186,7 +186,7 @@ inline void Blitter_32bppSSE4_Anim::Draw(const Blitter::BlitterParams *bp, ZoomL
 					const uint m1 = (byte) (mvX2 >> 16);
 					const uint r1 = remap[m1];
 					if (mvX2 & 0x00FF00FF) {
-						#define CMOV_REMAP(m_colour, m_colour_init, m_src, m_m) \
+#						define CMOV_REMAP(m_colour, m_colour_init, m_src, m_m) \
```

Comment:
I feel like this (and the other CMOV_REMAP define) should be moved so that it's not indented so much, it just looks really odd like this. Probably best move it to the top of the file, so it doesn't have to be indented at all

Reply:
I won't even pretend to know what the author's intentions were here. Somebody here must know how it works, though.

Reply:
Indeed. I guess it is highly specific to this particular block of code, so maybe just leave it where it is, but remove all the indentation

Reply:
How do you feel about jump labels starting in the first column? If it's bad; then don't look up 😎.

I'm seriously considering removing these particular files from the PR; let's just sweep it under the rug and pretend it never happened 😉.